ui/src/modules/events/store/services/event-overview.store.ts

```diff
@@ -117,12 +117,12 @@ export class EventOverviewStore extends BaseStore<EventOverviewStoreNs.State> {
             graphLoadingProcessing: ProcessingStoreNs.eventProcessingStart(this.state.graphLoadingProcessing)
         })
 
-        setTimeout(() => {
+        
             this.updateState({
```

Comment:
@TerranceKhumalo-absa I would recommend investigating why the developer called 2 times this.updateState (and the second time he used setTimeout) for a better understanding of how to refactor this code correctly.